relayer/src/worker.rs

```diff
@@ -120,7 +120,11 @@ pub fn spawn_worker_tasks<ChainA: ChainHandle, ChainB: ChainHandle>(
                     );
                     task_handles.push(packet_task);
 
-                    let link_task = packet::spawn_packet_worker(path.clone(), link);
+                    let link_task = packet::spawn_packet_worker(
+                        path.clone(),
+                        link,
+                        packets_config.clear_interval,
+                    );
```

Comment:
It seems like the `clear_interval` variable is only needed for being passed into `process_pending_txs`. If so, we can consider making things more explicit here by passing a `do_resubmit: bool`  argument to this method (instead of passing the `clear_interval: u64`, which is a bit clumsy).

This is an alternative to my comment above where I suggest we compute `do_resubmit` inside `process_pending_txs`. We could instead do that here.


```suggestion
                    // rationale...
                    let do_resubmit = if packets_config.clear_interval == 0 { true } else { false };
                    let link_task = packet::spawn_packet_worker(
                        path.clone(),
                        link,
                        do_resubmit,
                    );
```